doc/api/deprecations.md

```diff
@@ -95,12 +95,18 @@ methods, the `options.customFds` option is deprecated. The `options.stdio`
 option should be used instead.
 
 <a id="DEP0007"></a>
-### DEP0007: cluster worker.suicide
```

Comment:
Would it be possible to not include `worker.suicide` inside the error message but rather a reference to the documentation?

```
### DEP0007:  The api used is deprecated, please replace with worker.exitedAfterDisconnect. Please refer to documentation for more information.
```
In the extended documentation we could start with

```
Content warning: Self Harm.

Please click here to skip to the next entry.
```
We can then make "click here" an href to the next header